File: lab1.1/lab1.1_1/line.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab1._1_1
{
    class Line
    {
        private float xA, yA, xB, yB;

        public Line(float xa, float ya, float xb, float yb)
        {
            this.xA = xa;
            this.yA = ya;
            this.xB = xb;
            this.yB = yb;
        }

        public float dovzhina()
        {
             return (float)Math.Sqrt(Math.Pow(xB - xA, 2) + Math.Pow(yB - yA, 2));
        }

        public float kyt()
        {
            float vert = yB - yA;
            float hor = xB - xA;
            return (float)Math.Round(Math.Atan2(vert, hor)*(180/Math.PI), 2);
        }
    }
}

File: lab1.2plus/lab1.2plus/line.cpp
#include "Header.h"

float line::dovzhina() {
	return sqrt(pow(xB - xA, 2) + pow(yB - yA, 2));
}
float line::kyt() {
	float vert = yB - yA;
	float hor = xB - xA;
	return (float)round(atan2(vert, hor) * (180 / M_PI));
}